src/custom_nodes/east_ocr/config.json

```diff
@@ -63,7 +63,8 @@
                         "box_width_adjustment": "0.1",
                         "box_height_adjustment": "0.0",
                         "rotation_angle_threshold": "8",
-                        "debug": "true"
+                        "debug": "true",
```

Comment:
Maybe we should switch debug to off by default?
@dkalinowski

Reply:
I would not do it since this is only an example but still useful for reading logs in end to end tests (from validation team)

Reply:
On the other hand this default "true" makes performance testing have small overhead @atobiszei

Reply:
Without debug there is no information what happens inside execute function, so I agree that it is useful for logs reading and I would stick with "true" by default. However when I was testing all of CN locally the overhead that @dkalinowski mentioned was noticeable. Debug should be definitely set up to "false" in any performance tests.